examples/fluent-bit/grafana-loki/task-definition.json

```diff
@@ -0,0 +1,40 @@
+{
+    "family": "firelens-example-grafana-loki",
+    "taskRoleArn": "arn:aws:iam::XXXXXXXXXXXX:role/ecs_task_iam_role",
+    "executionRoleArn": "arn:aws:iam::XXXXXXXXXXXX:role/ecs_task_execution_role",
+    "containerDefinitions": [
+        {
+            "essential": true,
+            "image": "amazon/aws-for-fluent-bit:stable",
```

Comment:
I think there's an issue here because you link to 2 different loki plugins:
- https://grafana.com/docs/loki/latest/clients/fluentbit/
- https://docs.fluentbit.io/manual/v/1.9-pre/pipeline/outputs/loki

These are entirely different and have different options. You see to be using the one from the first link, the grafana custom plugin, which is not an upstream Fluent Bit plugin and requires that you use their custom image as shown in their docs here: https://grafana.com/docs/loki/latest/clients/fluentbit/

Have you actually tested this task def? 

Please re-write this example to be exclusively about only 1 of the loki plugins. Please make it clear that there are two.